### PR TITLE
Added autocomplete="off" to some password fields

### PR DESCRIPTION
**Pull request by FranseHamburger**

### CODE REVIEW

**Review by redfast00 — APPROVED** [review]

File: templates/session/login.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
         <div class="field">
           <label class="label">Password</label>
-          <input class="input is-medium" name="password" type="password" placeholder="Password" required />
+          <input class="input is-medium" name="password" type="password" placeholder="Password" required autocomplete="off" />
```
Comment (chvp) [inline templates/session/login.html:37]: Why would we not want autocomplete when signing in? I think we want to allow people to use password managers?

**Review by chvp — COMMENTED** [review]
See also: https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields

File: templates/session/login.html (hunk 1 above)
Comment (redfast00) [reply templates/session/login.html:37]: We got an email from UGent's automated vuln scanner
image

Comment (redfast00) [reply templates/session/login.html:37]: After looking at this a bit better; I don't really agree with the vuln scanner, and think the best course of action is emailing DICT to disable those plugins. Another garbage detection as an example: 
image

**Review by redfast00 — CHANGES_REQUESTED** [review]
Yeah, maybe we shouldn't do this; I think we should email UGent instead

**Comment by rien** [comment]
I am also against this, as this creates a less secure situation in my option.

**Pull request closed without merging**